src/pykeen/nn/functional.py

```diff
@@ -1188,7 +1181,7 @@ def cross_e_interaction(
     # relation interaction (notice that h has been updated)
     r = h * r
     # combination
-    x = activation(h + r + bias)
+    x = activation(h + r + bias.view(*(1 for _ in h.shape[:-1]), -1))
```

Comment:
_make_ones_like

unless this is getting ridiculous

Reply:
5309939